deploy-lvmo/deploy-hub.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -m
+
+# Load common vars
+source ${WORKDIR}/shared-utils/common.sh
+
+if ! ./verify_hub.sh; then
+    echo ">>>> Modify files to replace with pipeline info gathered"
+    echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    sed -i "s/CHANGEME/$OC_ODF_VERSION/g" manifests/03-LVMO-Subscription.yaml
+    sed -i "s/CATALOG_SOURCE/redhat-operators/g" manifests/03-LVMO-Subscription.yaml
+
+        echo ">>>> Deploy manifests to install LSO and LocalVolume: hub cluster"
+        echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+
+        oc --kubeconfig=${KUBECONFIG_HUB} apply -f manifests/01-Namespace.yaml
+        sleep 2
+        oc --kubeconfig=${KUBECONFIG_HUB} apply -f manifests/02-LVMO-OperatorGroup.yaml
+        sleep 2
+        oc --kubeconfig=${KUBECONFIG_HUB} apply -f manifests/03-LVMO-Subscription.yaml
+        sleep 5
+
+        echo ">>>> Waiting for subscription and crd on: hub cluster"
+        echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+        declare -a LVMOCRDS=("lvmclusters.lvm.topolvm.io" "lvmvolumegroupnodestatuses.lvm.topolvm.io" "lvmvolumegroups.lvm.topolvm.io")
+        for crd in ${LVMOCRDS[@]}; do
+            check_resource "crd" "${crd}" "Established" "openshift-local-storage" "${KUBECONFIG_HUB}"
+        done
+
+        echo ">>>> Render and apply manifest to deploy LVMCluster"
+        echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>"
+        oc --kubeconfig=${KUBECONFIG_HUB} apply -f manifests/04-LVMO-LVMOCluster.yaml
+        sleep 60
+
+        echo ">>>> Waiting for: LVMCluster"
+        echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+        timeout=0
+        ready=false
+        while [ "$timeout" -lt "1000" ]; do
+            if [[ $(oc get --kubeconfig=${KUBECONFIG_HUB} -n openshift-storage lvmcluster -ojsonpath='{.items[*].status.ready}') == "true" ]]; then
```

Comment:
This is relating to #538 
maybe we should remove the exit on error

Reply:
I will do this refactor in both places in a follow-up PR